cmd/admin-handlers.go

```diff
@@ -1925,7 +1925,7 @@ func getServerInfo(ctx context.Context, poolsInfoEnabled bool, r *http.Request)
 		Versions:     versions,
 		Usage:        usage,
 		Services:     services,
-		Backend:      backend,
+		Backend:      backend.(madmin.ErasureBackend),
```

Comment:
getServerInfo should change to start using `backend` from interface to `madmin.ErasureBackend` so that this reflection is not needed.